### ISSUE TITLE
Adding support for undocumented 6502 instructions

### ISSUE THREAD

**Issue by robmcmullen** [jefftranter/udis#1]
I'm interested in adding support for [undocumented/illegal opcodes in the 6502 ](http://nesdev.com/undocumented_opcodes.txt), but I wanted to check with you before I went marching off on my own. My idea was to add in the undocumented opcodes but use a new flag parameter in the opcodeTable dictionary, and the main program could then check whether or not to display the undocumented pcode based on a command line flag. If the command line arg isn't set to display those, then the disassembly would look just like it does now.

Did you have any ideas? I'll probably implement it that way unless you've already had some plans in mind.

**Comment by jefftranter** [comment]
Yes, that should work fine. If you come up with a patch I will take a look
and consider adding to my version.
